site/_components/authors-list.njk

```diff
@@ -7,7 +7,8 @@
   @param {string} microcopy.author.alt - String to describe the author's photo
 
  #}-->
-{% from 'author.njk' import author as authorMacro %} {% macro authorsList(authors, contributors, microcopy) -%}
+{% from 'author.njk' import author as authorMacro %} {% macro authorsList(label, authors, contributors, microcopy) -%}
+<span class="authors-list__label type--label">{{ label }}</span>
```

Comment:
See comment in `article.njk` about this maybe needing to be a heading of some sort

Reply:
Done, change to an `h4`